pages/_app.js

```diff
@@ -8,7 +8,7 @@ import { Redirect } from "@shopify/app-bridge/actions";
 import "@shopify/polaris/dist/styles.css";
 import "../css/styles.css";
 
-const apiKey = JSON.stringify(process.env.SHOPIFY_API_KEY);
+const apiKey = "cca7cc1aa184364b1476e9410ea38872";
```

Comment:
Umgebungsvariablen benutzen. Und den Key sollten wir nun einmal rotieren, da es ein öffentliches Repo ist. ^^
```suggestion
const apiKey = process.env.SHOPIFY_API_KEY;
```